### PR TITLE
Fix broken install link.

### PR DESCRIPTION
**Pull request by Shpetz**
Replace dead Azure CLI install link with a working one that points to the official docs.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5780
This is an **experimental preview** for @microsoft users.

File: doc/xplat_cli_migration.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ using NPM, upgrade with `npm upgrade -g azure-cli`.  If you used an installer,
 we recommend downloading the latest installer to upgrade.
 
 To install the Azure CLI 2.0, follow the steps for your preferred platform or
-environment on our [Installation Guide](https://github.com/Azure/azure-cli/blob/master/README.rst#installation).
+environment on our [Installation Guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
```
Comment (derekbekoe) [inline doc/xplat_cli_migration.md]: Please remove `?view=azure-cli-latest` from the URL.
Doc team would rather the server redirect to the appropriate view.

**Comment by Shpetz** [comment]
Updated the link as requested.

**Comment by derekbekoe** [comment]
@Shpetz The PR was made to the wrong branch. I have submitted a new PR with your commits to the correct dev branch.

**Comment by derekbekoe** [comment]
https://github.com/Azure/azure-cli/pull/6054

**Pull request merged**